src/commands/Movie.js

```diff
@@ -0,0 +1,69 @@
+const { Command } = require('@dyno.gg/dyno-core');
+
+class Movie extends Command {
+	constructor(...args) {
+		super(...args);
+
+        this.aliases        = ['movie'];
+        this.module         = 'Fun';
+        this.description    = 'Get info on a movie.';
+        this.usage          = 'movie [movie name]';
+        this.example        = 'movie Deadpool';
+        this.cooldown       = 3000;
+        this.expectedArgs   = 1;
+	}
+
+	async execute({ message, args }) {
+		const MovieDB = require('movieDB')(this.config.movieDBKey);
+		args = args.join(' ');
+
+		MovieDB.searchMovie({
+			query: args,
+			page: 1,
+			include_adult: false
+		}, (err, res) => {
+			if (err) return await this.error(msg.channel, 'Uh oh! Something went wrong with that query!');
+
+			res = res.results[0];
+
+			let relDate = (new Date(res.release_date)).toLocaleDateString('en-US', {
+				weekday: 'long',
+				year: 'numeric',
+				month: 'long',
+				day: 'numeric'
+			});
+
+			return await this.sendMessage(msg.channel, {
```

Comment:
This shouldn't return await, the command has to return a Promise, returning `this.sendMessage()` would be good.